api/server/router/swarm/cluster_routes.go

```diff
@@ -132,7 +132,7 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 
 	id, err := sr.backend.CreateService(service, encodedAuth)
 	if err != nil {
-		logrus.Errorf("Error creating service %s: %v", id, err)
+		logrus.Errorf("Error creating service: %v", err)
```

Comment:
Yes, but I think it is better than id which is always empty.
Maybe we can add image name as well for understandability.
How about this?
```go
logrus.Errorf("Error creating service %s from image %s: %v", service.Name, service.TaskTemplate.ContainerSpec.Image, err)
```